### PR TITLE
[FIX] mail, web: correctly handle mounting operation cancelled error

### PR DESCRIPTION
**Pull request by alexkuhn**

### CODE REVIEW

File: addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js [hunk 1]
```diff
@@ -111,7 +111,14 @@ FormRenderer.include({
...
     async _mountChatterContainerComponent() {
-        await this._chatterContainerComponent.mount(this._chatterContainerTarget);
+        try {
+            await this._chatterContainerComponent.mount(this._chatterContainerTarget);
+        } catch (error) {
+            if (error.message === "Mounting operation cancelled") {
+                // do nothing
+            }
+            throw error;
+        }
```
Comment (seb-odoo) [inline addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js]:
```suggestion
        try {
            await this._chatterContainerComponent.mount(this._chatterContainerTarget);
        } catch (error) {
            if (error.message !== "Mounting operation cancelled") {
                throw error;
            }
        }
```
Isn't that what we want?
The catch seems useless if we always throw again.

File: addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js [hunk 2]
```diff
@@ -141,17 +148,24 @@ FormRenderer.include({
...
+    async _updateChatterContainerComponent() {
         const props = this._makeChatterContainerProps();
-        this._chatterContainerComponent.update(props);
+        try {
+            await this._chatterContainerComponent.update(props);
+        } catch (error) {
+            if (error.message === "Mounting operation cancelled") {
+                // do nothing
+            }
+            throw error;
```
Comment (seb-odoo) [inline addons/mail/static/src/messaging/widget/form_renderer/form_renderer.js]: same

**Review by seb-odoo — CHANGES_REQUESTED** [review]

**Comment by alexkuhn** [comment]
just applied suggested changes

**Comment by seb-odoo** [comment]
Ok, looks better. So if you remove the await stuff, with the previous version it crashes, and the new version it works? Because the diff was just doing nothing otherwise.

**Comment by alexkuhn** [comment]
It no longer crashes if we use `.catch()` in update, without async. I
prefer async await for the nicer syntax.

**Pull request merged**